standalone/OAST_Listen_Burp_Colaborator.py

```diff
@@ -0,0 +1,88 @@
+# Version 1.0
+# @author RUFFENACH Timothée
+# Get OAST burp colaborator.
+
+from javax.swing import JFrame, JPanel, JComboBox, JOptionPane,JFileChooser,JOptionPane
+import urllib2
+import json
+import sys
+import base64
+import time
+
+
+global biid
+
+def main():
+        global biid 
+        biid = getString("what is your biid ?")
```

Comment:
```suggestion
        biid = getString("What is your biid ?")
```

Will "biid" be clear to a user because I have no idea what it is.

Reply:
The biid is a token of burp collaborator protection, i wireshark the token name is biid. Look link https://infosecwriteups.com/setup-your-private-burp-collaborator-for-ssrf-xxe-fd6cf01c8ca

Reply:
I don't understand the meaning of biid but I know that it stores the user's secret key.
I replaced What is your biid ? by What is your secret key (biid) ? 
The capital letters have been put.